packages/client/locales/cn/account.json

```
@@ -0,0 +1,52 @@
+{
+  "Account": {
+    "account": "帐户",
+    "accountInfo": "账户信息",
+    "accountSettings": "账户设置",
+    "passwordSettings": "密码设置",
+    "appearanceSettings": "外观设置",
+    "connections": "连接",
+    "connectDiscord": "连接不和谐帐户",
+    "disconnectDiscord": "断开不和谐帐户",
+    "disconnectText": "这将删除通过Discord登录的功能.",
+    "disabledDisconnectText": "启用“禁止常规登录”时，您无法断开Discord连接。要删除您的帐户，请联系管理员",
+    "connectText": "这将增加通过Discord登录的功能。",
+    "disconnecting": "断开。。。",
+    "currentPassword": "当前密码",
+    "newPassword": "新密码",
+    "2fa": "双因素身份验证",
+    "confirmNewPassword": "确认新密码",
+    "scanQRCode": "扫描此 QR 码以将其添加到身份验证器应用",
+    "verifyCode": " 通过输入验证器应用中的代码进行验证",
+    "disable": "禁用",
+    "enable2FA": "启用双因素身份验证",
+    "disable2FA": "禁用双因素身份验证",
+    "next": "下一个",
+    "verify": "验证",
+    "enable2faSuccess": "已成功启用双因素身份验证",
+    "disable2faSuccess": "已成功禁用双因素身份验证",
+    "darkTheme": "黑暗主题",
+    "statusView": "状态视图",
+    "tableAlignment": "表操作对齐方式",
+    "sounds": "听 起来",
+    "panicButton": "紧急按钮",
+    "signal100": "信号 100",
+    "stopRoleplay": "停止角色扮演",
+    "addedToCall": "已添加到呼叫",
+    "statusUpdate": "状态更新",
+    "incomingCall": "来电",
+    "dotColor": "点阵颜色",
+    "fullRowColor": "整行颜色",
+    "userApiToken": "用户 API 令牌",
+    "reGenerateToken": "重新生成令牌",
+    "discord": "不和",
+    "steam": "蒸汽",
```

These are names of companies, these should stay the same.
```suggestion
    "discord": "Discord",
    "steam": "Steam",
```

---

ok I modified it and uploaded it